scripts/deploy.ts

```diff
@@ -0,0 +1,30 @@
+// We require the Hardhat Runtime Environment explicitly here. This is optional
```

Comment:
this file can be removed

Reply:
we will work on the deploy script in a separate task